vision.md

```diff
@@ -1,52 +1,41 @@
-Stated simply...
+## Stated simply...
```

Comment:
All of our documents have a title whereas the title of this document is the filename. I think we should add a title `NUnit Vision Statement` and move this down to a subtitle.